src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationHandler.cs

```diff
@@ -58,15 +56,13 @@ private Task<AuthenticationTicket> EnsureCookieTicket()
                 var claim = ticket.Principal.Claims.FirstOrDefault(c => c.Type.Equals(SessionIdClaim));
                 if (claim == null)
                 {
-                    Logger.LogWarning(@"SessionId missing");
-                    return null;
+                    return AuthenticateResult.Fail("SessionId missing");
                 }
                 _sessionKey = claim.Value;
                 ticket = await Options.SessionStore.RetrieveAsync(_sessionKey);
                 if (ticket == null)
                 {
-                    Logger.LogWarning(@"Identity missing in session store");
-                    return null;
+                    return AuthenticateResult.Fail("Identity missing in session store");
```

Comment:
We went from Warning to Info as part of this change too right?  I don't remember what the guidance is for which we should use